recipes/unicorn-binance-websocket-api/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "unicorn-binance-websocket-api" %}
+{% set version = "1.40.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unicorn-binance-websocket-api-{{ version }}.tar.gz
+  sha256: 4a2d00830e051ae4b51be117b435f5a1e73a5d3a22a31a8c14fb3ce276edf213
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - cheroot
+    - colorama
+    - flask
+    - flask-restful
+    - psutil
+    - python >=3.7
+    - requests
+    - ujson
+    - unicorn-fy
+    - websocket-client
+    - websockets ==10.2
+  run:
+    - cheroot
+    - colorama
+    - flask
+    - flask-restful
+    - psutil
+    - python >=3.7
+    - requests
+    - ujson
+    - unicorn-fy
+    - websocket-client
+    - websockets ==10.2
+
+test:
+  imports:
+    - tools
```

Comment:
This means that you're installing a separate package called tools. Is that on purpose?

Reply:
no, thats wrong, thanks